src/app/FakeLib/NuGet/NugetHelper.fs

```diff
@@ -19,6 +19,10 @@ type NugetFrameworkReferences =
     { FrameworkVersion : string
       References : NugetReferences }
 
+type NugetFrameworkAssemblyReferences =
```

Comment:
I guess I could've reused the existing `NugetFrameworkReferences` type and grouped framework assembly references on `FrameworkVersion` instead. What do you think?